libflux/flux/src/bin/fluxdoc.rs

```diff
@@ -0,0 +1,94 @@
+#![cfg_attr(feature = "strict", deny(warnings))]
+
+//use std::collections::HashMap;
+use flux::docs::{self, DocPackage};
+use std::fs::{self, File};
+use std::io::Write;
+use std::path::{Path, PathBuf};
+use structopt::StructOpt;
+use tera::{Context, Tera};
+
+#[macro_use]
+extern crate lazy_static;
+
+//use flux::ast;
+//use flux::docs;
+//use flux::parser::Parser;
+//use flux::semantic;
+//use flux::semantic::types::{PolyType, TvarKinds};
+
+#[derive(Debug, StructOpt)]
+struct Args {
+    // The root path of packages for which to generate documentation
+    #[structopt(parse(from_os_str), long)]
+    pkg: PathBuf,
+    // The name of the file to write the documentation JSON data
+    #[structopt(parse(from_os_str), long, default_value = "")]
+    json: PathBuf,
+    // The name of the directory into which to write the documentation html files
+    #[structopt(parse(from_os_str), long, default_value = "")]
+    html: PathBuf,
+}
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let args = Args::from_args();
+    let pkg = docs::walk_pkg(&args.pkg, &args.pkg)?;
+
+    if args.json != Path::new("") {
+        let f = File::create(args.json)?;
+        serde_json::to_writer(f, &pkg)?;
+    }
+
+    if args.html != Path::new("") {
+        write_home(&args.html)?;
+        return write_html(&args.html, &pkg);
+    }
+    Ok(())
+}
+
+lazy_static! {
+    static ref TEMPLATES: Tera = {
+        let mut tera = match Tera::new("fluxdoc/templates/*.html") {
```

Comment:
Looks like this path needs to be updated.

Reply:
Ah, right. It would just be `"flux/templates/*.html"`, correct?